packages/ui-extensions/src/surfaces/customer-account/api/order-confirmation/order-confirmation.ts

```diff
@@ -0,0 +1,22 @@
+import type {StatefulRemoteSubscribable} from '@remote-ui/async-subscription';
+
+export interface OrderConfirmation {
+  order: {
+    /**
+     * The globally-uniqueID of the OrderConfirmation. This will be the ID of the Order once successfully created.
+     */
+    id: string;
+  };
+  /**
+   * A randomly generated alpha-numeric identifier for the order.
+   * This number should always be present, but 2023 orders predate confirmation numbers and may not have it.
+   */
+  number?: string;
+}
+
+export interface OrderConfirmationApi {
```

Comment:
Where is this used ?

Reply:
The customer account targets needs to be updated in the customer account surface as well if we want to expose this, not just in checkout where they will be deprecated.